.cargo/config

```diff
@@ -1,2 +1,13 @@
 [alias]
 regen-protobufs = ["run", "--bin", "protobuf-gen", "tools/protobuf_files.toml"]
+dev-install = ["install", "asdev"]
+all_tests = ["asdev", "test"]
+rust_tests = ["asdev", "test_rust"]
+verify_env = ["asdev", "verify_env"]
+smoke_android = ["asdev", "smoke_android"]
+smoke_fenix = ["asdev", "smoke_fenix"]
+smoke_ios = ["asdev", "smoke_ios"]
```

Comment:
let's remove the smoke testing for now and make it simpler. I filed an issue https://github.com/a-s-dev/asdev/issues/14